core/app/models/spree/order.rb

```diff
@@ -787,7 +787,7 @@ def after_cancel
     end
 
     def send_cancel_email
-      OrderMailer.cancel_email(self.id).deliver_later
+      OrderMailer.cancel_email(self).deliver_later
```

Comment:
@jhawthorn - this won't play nicely with anyone (like Bonobos) that's using [ResqueMailer](https://github.com/zapnap/resque_mailer) since it won't hit the new Rails 4.2 GlobalID stuff and the order will be passed into queues as an ActiveRecord object instead of as an integer.  I know that's caused problems for us in the past.

Bonobos can and probably should stop using ResqueMailer anyway (it's a bit broken on Rails 4.2 anyway) but would be nice if there were a temporary workaround that didn't require us to monkey-patch `Order#send_cancel_email`. Any ideas? Or perhaps at least a warning in the changelog that people need to get rid of ResqueMailer (and perhaps others like it, if there are any)?